examples/userapi/05_conditional_dimension.ipynb

```diff
@@ -0,0 +1,882 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# `ConditionalDimension` tutorial"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial explains how to create equations that only get executed under given conditions.\n",
+    "\n",
+    "As in all our tutorials, we begin by specifying some parameters.\n",
+    "For our examples we will use `Function`s defined on 2D grids and their data values are going to be updated according to prescribed conditions."
```

Comment:
I'd also drop this sentence (@rhodrin )

Reply:
Dropped. I think however that first cell is now getting very `steep`